service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/EmailAddress.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class EmailAddress {
```

Comment:
I also have email address, damian had commented it should be email but email didnt work for me, did you try calling it just email?

Reply:
I'll give it a go...Change sounds good

Reply:
EmailAddress might be better. It's consistent with the email type defined in confluence.

Reply:
I agree, @geek-soft commented on mine with the same but i agree with you

Reply:
@geek-soft @swalker125 thoughts?

Reply:
I am fine with EmailAddress